SmartDeviceLink-iOS/SmartDeviceLink/SDLProxyListener.h

```diff
@@ -121,5 +124,8 @@
 - (void)onUnregisterAppInterfaceResponse:(SDLUnregisterAppInterfaceResponse *)response;
 - (void)onUnsubscribeButtonResponse:(SDLUnsubscribeButtonResponse *)response;
 - (void)onUnsubscribeVehicleDataResponse:(SDLUnsubscribeVehicleDataResponse *)response;
+- (void)onAlertManeuverResponse:(SDLAlertManeuverResponse*) request;
+- (void)onDialNumberResponse:(SDLDialNumberResponse*) request;
+- (void)onSendLocationResponse:(SDLSendLocationResponse*) request;
```

Comment:
The rest of the proxyListener methods are in alphabetical order, these should be as well.